hack/tls/gen_certs.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
```

Comment:
This file, serving as reference, should be moved at the top level in the Scripts directory.
User then should be able to modify any of the parameters as needed.

Reply:
I dont have any preference. This directory is also meant for keeping scripts etc.

Reply:
Yeah, we can leave this directory for CI/CD related scripts. And use the Scripts directory at the root for deployment related scripts.